hdf5-warning.patch

```diff
@@ -1,80 +1,74 @@
-From 55f7415d779b19b1d0b3d8db57fa5260441d2eb0 Mon Sep 17 00:00:00 2001
-From: Maureen Jean <mjean@boro-70.boro.hpdd.intel.com>
-Date: Fri, 17 Jul 2020 21:10:32 +0000
+From e74d6ae412fd2246784bc517727ce4b36a75e6e7 Mon Sep 17 00:00:00 2001
```

Comment:
Isn't this patch removed from the spec?  If so, the file should be completely removed from the repo.

Reply:
it has been removed